reinforcement_learning/custom-recipes/Choose the environment, the agent and train it/recipe.json

```diff
@@ -0,0 +1,256 @@
+// This file is the descriptor for the Custom code recipe Choose the environment, the agent and train it
+{
+    // Meta data for display purposes
+    "meta" : {
+        "label" : "Training",
+        "description" : "Choose the environment, the agent and train it",
```

Comment:
Again, think outside of the context you know, understand who will read that and when.